### DISCUSSION TITLE
Get payment info from mercadopago api

### DISCUSSION THREAD

**Discussion by Pacocc97** [mercadopago/sdk-react#50] [Q&A]
Hello my people. 
How do you get the information of a specific payment from mercadopago api?
I'm aware that I need to make a get request from "https://api.mercadopago.com/v1/payments/{payment id}", but I'm have some trouble fetching from my front end app, due to mercadopago CORS policy!!

Has anyone else had the same problem??

**Comment by ghost** [comment]
Answered in https://github.com/mercadopago/sdk-react/discussions/51.
I will close this discussion.
